Geospatial.md

```diff
@@ -94,6 +94,36 @@ Bounding box is defined as the thrift struct below in the representation of
 min/max value pair of coordinates from each axis. Note that X and Y Values are
 always present. Z and M are omitted for 2D geospatial instances.
 
+Writers should follow the guidelines below when calculating bounding boxes in
+the presence of invalid values. An invalid geospatial value refers to any of 
+the following: `NaN`, `null`, `does not exist` (e.g., LINESTRING EMPTY), or 
```

Comment:
It seems worth providing concrete example for each case? For example, I still don't understand what does `null` mean here. Is it a null binary value in Parquet, or a null value in WKB? We can add a section below named `Invalid geospatial value` and link it here.

Reply:
I just created a separate section for `invalid geospatial values`. Please check.